api/src/main/java/org/openmrs/module/idgen/IdgenModuleActivator.java

```diff
@@ -31,6 +33,7 @@
 	@Override
 	public void started() {
 		IdgenTask.setEnabled(true);
+		createLocationPrefixAttributeType();
```

Comment:
You should not be creating Location Attributes on idgen module startup.  This is not a core function of the idgen module and might have other implications.  If an implementer wants to use the LocationBasedPrefixProvider, then can take responsibility over ensuring that the correct LocationAttributeType and LocationAttributes exist.